tests/iostability/test_iostability_degraded_path.py

```diff
@@ -68,23 +68,41 @@ def setup_class(cls):
         cls.s3t_obj = S3TestLib()
         cls.iolib = IOStabilityLib()
         cls.test_completed = False
+        cls.duration_in_days = os.getenv("DURATION_OF_TEST_IN_DAYS",
+                                         cls.test_cfg['happy_path_duration_days'])
+
+        cls.clients = os.getenv("CLIENT_SESSIONS_PER_WORKER_NODE", 0)
+        if cls.clients == 0:
+            if cls.setup_type == 'HW':
+                cls.clients = cls.test_cfg['sessions_per_node_hw']
+            else:
+                cls.clients = cls.test_cfg['sessions_per_node_vm']
+        cls.mail_id = os.getenv("MAIL_ID", None)
+        cls.start_time = datetime.datetime.now()
+        cls.mail_notify = MailNotification(sender="priyanka.borawake@seagate.com",
```

Comment:
Does it not need to be flexible other than your email in code? In that case init at the top would be a better idea.

Reply:
Sender won't be of much concern in this case. Receiver mail id is taken as input to keep this flexible. 
We can also use some mail alias which can be created for sending notification mail.

Reply:
Better to use some alias or generic email. DO NOT expose any personal info in the code. :)

Reply:
Yes, modified the code to accept sender mail also as input.